### PR TITLE
Slingshot ammo base class

### PR DESCRIPTION
**Pull request by CrazyTobi**

### CODE REVIEW

File: Source/SteveSloth/SlingshotAmmoBaseClass.h [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	virtual void BeginPlay() override;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Name")
+	FString Name;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Damage")
+	float Damage;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Damage")
+	bool AreaOfEffect;
```
Comment (Jeff-Moreau) [inline Source/SteveSloth/SlingshotAmmoBaseClass.h:48]: Should also have an AOE damage amount
and the Radius of how far out the AOE goes from the center

File: Source/SteveSloth/SlingshotAmmoBaseClass.h [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Damage")
+	bool DamageOverTime;
```
Comment (Jeff-Moreau) [inline Source/SteveSloth/SlingshotAmmoBaseClass.h:51]: Should also have a DOT damage amount
add also the amount of time the DOT will tick for

**Review by Jeff-Moreau — CHANGES_REQUESTED** [review]

**Review by Jeff-Moreau — APPROVED** [review]
Looks Good

**Review by Guysus — COMMENTED** [review]
Looks good to me

**Pull request merged**